### DISCUSSION TITLE
I boot the live iso and i am asked for user credentials

### DISCUSSION THREAD

**Discussion by RaptaG** [Nitrux/nitrux-bug-tracker#130] [Q&A]
What should I fill it with?

**Comment by UriHerrera** [comment]
This has already been fixed.

[Calamares cannot enable autologin in SDDM for the Plasma Wayland session #105](https://github.com/Nitrux/nitrux-bug-tracker/issues/105).
[Live ISO with Plasma Desktop requires manual login (does not autologin to desktop) #119](https://github.com/Nitrux/nitrux-bug-tracker/issues/119).
